examples/todomvc/index.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import App from './containers/App';
+import 'style!raw!./node_modules/todomvc-app-css/index.css';
```

Comment:
Won't just `style!raw!todomvc-app-css/index.css` work? This `node_modules` thing looks weird.